core-server/src/main/java/org/orbisgis/orbisserver/control/xml/GetCapabilitiesController.java

```diff
@@ -101,8 +103,8 @@ public Result displayXML(@Parameter("service") String service, @Parameter("versi
                  if(request.equals("GetCapabilities")){
                    try {
                      indexController.GetXMLFromGetCapabilities();
-                   } catch (JAXBException ignored) {
-                     LOGGER.error(I18N.tr("Unable to parse the incoming xml. \nCause : {0}.", ignored.toString()));
+                   } catch (JAXBException e) {
+                     LOGGER.error(I18N.tr("Unable to parse the incoming xml. \nCause : {0}.", e.getMessage()));
```

Comment:
The exception should be return in the same way as the xml content to allow the client to receive the exception.